parent/pom.xml

```diff
@@ -80,8 +80,12 @@
 		<jbosstools-browsersim-site>${jbosstools-site-baseurl}/jbosstools-browsersim_${jbosstools-site-stream}/${jbosstools-site-suffix}</jbosstools-browsersim-site>
 		<jbosstools-webservices-site>${jbosstools-site-baseurl}/jbosstools-webservices_${jbosstools-site-stream}/${jbosstools-site-suffix}</jbosstools-webservices-site>
 
+		<!-- TODO: when changing these stable or integration URLs, be sure to ALSO change the URLs in download.jboss.org/jbosstools/builds/staging/_composite_/core/* or else composite builds will fail and no aggregates will happen in Jenkins! -->
 		<!-- JBIDE-18869: use specific releases composited together from latest at eclipse, mirroned onto dl.jb.org -->
-		<eclipse-thym-site>http://downloads.jboss.org/jbosstools/updates/integration/luna/core/thym/</eclipse-thym-site>
+		<eclipse-thym-site>http://downloads.jboss.org/jbosstools/updates/stable/luna/core/thym/</eclipse-thym-site>
```

Comment:
do we consider 0.1 the *stable* one for JBoss Tools ? I would have expected it to be outside of stable until we actually release it as part of JBoss Tools ?